src/Compilers/CSharp/Portable/Operations/CSharpOperationFactory.cs

```diff
@@ -1292,6 +1294,19 @@ private ICoalesceOperation CreateBoundNullCoalescingOperatorOperation(BoundNullC
             return new LazyCoalesceExpression(expression, whenNull, valueConversion, _semanticModel, syntax, type, constantValue, isImplicit);
         }
 
+        private IOperation CreateBoundNullCoalescingAssignmentOperatorOperation(BoundNullCoalescingAssignmentOperator boundNode)
```

Comment:
>OperatorOperation [](start = 62, length = 17)

`OperatorOperation`?

Reply:
Yep. The format in this file is `Create` + `BoundNodeName` + `Operation`.

---
In reply to: [207689657](https://github.com/dotnet/roslyn/pull/29063#discussion_r207689657) [](ancestors = 207689657)